tests/kernel/msgq/msgq_api/src/test_msgq_contexts.c

```diff
@@ -88,6 +88,36 @@ static void msgq_thread(struct k_msgq *pmsgq)
 	purge_msgq(pmsgq);
 }
 
+static void thread_entry_overflow(void *p1, void *p2, void *p3)
+{
+	u32_t rx_buf[MSGQ_LEN];
+
+	k_msgq_get(p1, &rx_buf[0], K_FOREVER);
+	k_msgq_get(p1, &rx_buf[1], K_FOREVER);
```

Comment:
check for return value

Reply:
okay. Thanks for review @nashif . Changes has been pushed